src/localization/ru/learn.json

```diff
@@ -125,7 +125,7 @@
   "steps.withoutSpaceCharacter.title": "Кроме символа пробела `\\S`",
   "steps.withoutSpaceCharacter.description": "`\\S` используется для поиска непробельных символов.",
 
-  "steps.lookarounds.title": "Lookarounds",
+  "steps.lookarounds.title": "Lookarounds (Позиционные проверки)",
```

Comment:
This title can change this way.
```json
"steps.lookarounds.title": "Позиционные проверки",
```

Because the original titles are already shown.
![image](https://user-images.githubusercontent.com/7966133/145086567-01d1ed98-fda5-4db1-ba8f-247cc53ed92c.png)

Reply:
I see, but Haosik translation variation extends localization and it's not bad.

Reply:
> I see, but Haosik translation variation extends localization and it's not bad.

You may be right. ✨